src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageWithPropertiesExtensions.cs

```diff
@@ -65,14 +66,15 @@ public static class LoggerConfigurationAzureTableStorageWithPropertiesExtensions
 			bool writeInBatches = false,
 			TimeSpan? period = null,
 			int? batchPostingLimit = null,
-			string additionalRowKeyPostfix = null)
+            string additionalRowKeyPostfix = null,
+            Func<LogEvent, string, string> rowKeyFunc = null)
```

Comment:
I wonder if a better variable name might be something like `rowKeyGenerator` or something else that is more intention revealing?

Also, `Func<LogEvent, string, string>` - it's not immediately clear what the string property is, I wonder if we should create a named delegate or something else?

Actually, I'm a bit confused because it looks like `AzureBatchingTableStorageSink` takes a `Func<LogEventEntity, string, int, string>` not a `Func<LogEvent, string, string>`?